HHS/OPRE-OPS
# Ops 1036/task 1307

## What changed

Enforce a subset of the rules on the frontend as well as add toolttip for deleting an agreement.

## Issue

#1307 

## How to test

1. Goto agreements list
2. ❌ attempt to delete the first agreement, it should not work
3. ✅ add yourself as teamMember or ProjectOfficer and delete // it should work
4. create a new agreement
5. ✅ attempt to delete the agreement you created, it should work

## Screenshots

https://github.com/HHS/OPRE-OPS/assets/4629398/fbd6a442-8f8a-4e00-a44e-9dbeef018487
